onos-config/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 name: onos-config
-version: 1.6.4
+version: 1.6.5
 kubeVersion: ">=1.17.0"
-appVersion: v0.10.18
+appVersion: v0.10.19
```

Comment:
image tag also?

Reply:
It's done by a template

Reply:
https://github.com/onosproject/onos-helm-charts/blob/3d42ffe818527da564ca3e9d68bec2c14d948051/onos-config/values.yaml#L26